test/e2e/network/topology_hints.go

```diff
@@ -124,12 +152,12 @@ var _ = common.SIGDescribe("Feature:Topology Hints", func() {
 			}
 		}
 
-		nodeList, err := c.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
-		framework.ExpectNoError(err)
 		nodesByZone := map[string]string{}
```

Comment:
I don't think we need this any more

Reply:
Actually I think it is needed because `podsByZone` still needs it to set up.